providers/flagd/pkg/service/in_process/service.go

```diff
@@ -90,11 +90,8 @@ func (i *InProcess) Init() error {
 		return err
 	}
 
+	initOnce := parallel.Once{}
 	syncInitSuccess := make(chan interface{})
-	readyOnce := parallel.OnceFunc(func() {
```

Comment:
`OnceFunc` is from Go 1.21, making the provider incompatible with older go versions

Reply:
bummer, I like that one, but I guess it's just syntax sugar.

Reply:
Agree, it's straightforward but breaks when used with a lower go version